oval-schemas/oval-definitions-schema.xsd

```diff
@@ -1103,6 +1115,50 @@
             -->
         </xsd:restriction>
     </xsd:simpleType>
+    <xsd:simpleType name="SortEnumeration">
+        <xsd:annotation>
+            <xsd:documentation>The SortEnumeration simple type defines basic sorting operations.  Currently 'document', 'lexical', 'numeric' and 'natural' are defined.</xsd:documentation>
+        </xsd:annotation>
+        <xsd:restriction base="xsd:string">
+            <xsd:enumeration value="document">
+                <xsd:annotation>
+					<xsd:documentation>No sorting is performed, the values will remain in the order they came in as, and match the order in which the referenced system characterics will appear in the OVAL results XML document.</xsd:documentation>
+                </xsd:annotation>
+			</xsd:enumeration>	
+            <xsd:enumeration value="lexical">
+			    <xsd:annotation>
+                    <xsd:documentation>Sort alphabetically, useful for pure string lists.</xsd:documentation>
+                </xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="numeric">
+			    <xsd:annotation>
+                    <xsd:documentation>Sort numerically, useful for lists of numbers or lists of integers or floats, but will cause errors if string data is present.</xsd:documentation>
+                </xsd:annotation>
+			</xsd:enumeration>
+			<xsd:enumeration value="natural">
+			    <xsd:annotation>
+                    <xsd:documentation>Sort like PHP's natsort https://www.php.net/manual/en/function.natsort.php</xsd:documentation>
```

Comment:
Natural sort order is a pretty established thing, with implementations in most programming languages and on most operating systems.  So, I think this might be a better external URL to reference: https://en.wikipedia.org/wiki/Natural_sort_order

Reply:
Sounds good.